src/ch/imedias/rsccfx/view/SupporterAttributesDialog.java

```diff
@@ -64,7 +84,8 @@ private void layoutForm() {
     gridPane.setHgap(20);
     gridPane.setVgap(10);
     gridPane.setPadding(new Insets(25, 25, 25, 25));
-    dialog.setResizable(true);
+    gridPane.autosize();
```

Comment:
is it really necessary to call this? as far as I know, it is the standard behavior of a gridpane already